besu/src/main/java/org/hyperledger/besu/controller/CliqueBesuControllerBuilder.java

```diff
@@ -57,7 +56,7 @@
 
   @Override
   protected void prepForBuild() {
-    localAddress = Util.publicKeyToAddress(nodeKeys.getPublicKey());
+    localAddress = Util.publicKeyToAddress(nodeKey.getPublicKey());
```

Comment:
nit: `getNodeKey` instead of accessing super class member variable.

Reply:
trying to leave as much untouched as possible - so wanted to leave this as is

Reply:
fair enough.